Mathlib/Data/Set/Pairwise/Basic.lean

```diff
@@ -3,22 +3,22 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl
 
-! This file was ported from Lean 3 source module data.set.pairwise
-! leanprover-community/mathlib commit 207cfac9fcd06138865b5d04f7091e46d9320432
+! This file was ported from Lean 3 source module data.set.pairwise.basic
```

Comment:
Ugh, renaming this file was a mistake; the port dashboard doesn't track renames well.